cmd/test/test.go

```diff
@@ -171,6 +177,13 @@ func newTestCommand() *cobra.Command {
 					if err != nil {
 						return appstudioErrorHandler(runner.NoFail, "output results", err)
 					}
+
+					if outputFilePath != "" {
+						err := os.WriteFile(outputFilePath, reportOutput, 0600)
+						if err != nil {
+							return fmt.Errorf("creating output file: %w", err)
+						}
+					}
 					fmt.Printf("%s\n", reportOutput)
```

Comment:
Just noticed we're not using `cmd.OutOrStdout()` here, we should, could be in a followup PR as this wasn't added here.

Reply:
I tossed it in!